README.md

```diff
@@ -1,37 +1,129 @@
 # Node-DuckDB
 
+###### [API](https://github.com/deepcrawl/node-duckdb/docs/api/node-duckdb.md) | [Code Of Conduct](https://github.com/deepcrawl/node-duckdb/docs/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/deepcrawl/node-duckdb/docs/CONTRIBUTING.md) | [Developing](https://github.com/deepcrawl/node-duckdb/docs/DEVELOPING.md)
+
+> Production ready DuckDB Node.js library written in TypeScript.
+> [<img src="https://www.deepcrawl.com/wp-content/themes/deepcrawl/images/deepcrawl-logo.svg" height="200" width="300" align="right">](https://www.deepcrawl.com/)
```

Comment:
We can probably drop the quotes here. If anything, just bold the text.

Reply:
Hmmm I actually think the quote looks pretty (I've taken the idea from puppeteer https://github.com/puppeteer/puppeteer/blob/main/README.md )

Reply:
Cool, np